external-crates/move/move-compiler/src/diagnostics/mod.rs

```diff
@@ -439,6 +439,18 @@ impl WarningFilters {
             },
         }
     }
+
+    pub fn unused_warnings_filter() -> Self {
```

Comment:
This should probably be something like `unused_function_and_struct_filter`, especially since by the name `unused_warnings_filter` to me would feel like a warning filter with the category filter of `UnusedItem`